### PR TITLE
[IMP] l10n_*: Update manifests to redirect to own documentation

### PR DESCRIPTION
**Pull request by moerradi**
Removing external links from localization manifests and redirect to our own documentation. Ensure that users can learn about our standard localization modules from a source of information that we have authorship on.

Task-Id: 3248632

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_at/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
     'name': 'Austria - Accounting',
     'version': '3.0',
     'author': 'WT-IO-IT GmbH, Wolfgang Taferner',
```
Comment (qdp-odoo) [inline addons/l10n_at/__manifest__.py:5]:
```suggestion
    'author': 'WT-IO-IT GmbH, Wolfgang Taferner (https://www.wt-io-it.at)',
```
Please keep the link to the author's website. I feel like it's a nice way to thank them for their contributions to let them keep that visibility

Same applies for every localization we had a  link to the author's website

**Review by qdp-odoo — CHANGES_REQUESTED** [review]
see my comment on l10n_at

**Comment by moerradi** [comment]
on it

File: addons/l10n_cr/__manifest__.py [hunk 2]
```diff
@@ -2,8 +2,8 @@
 {
     'name': 'Costa Rica - Accounting',
     'url': 'https://github.com/CLEARCORP/odoo-costa-rica',
-    'author': 'ClearCorp S.A.',
-    'website': 'http://clearcorp.co.cr',
+    'author': 'ClearCorp S.A. (http://clearcorp.co.cr)',
```
Comment (qdp-odoo) [inline addons/l10n_cr/__manifest__.py:5]: to drop

File: addons/l10n_cn/__manifest__.py [hunk 3]
```diff
@@ -3,9 +3,9 @@
     'name': 'China - Accounting',
     'version': '1.8',
     'category': 'Accounting/Localizations/Account Charts',
-    'author': 'www.openerp-china.org',
+    'author': 'www.openerp-china.org (http://openerp-china.org)',
```
Comment (qdp-odoo) [inline addons/l10n_cn/__manifest__.py:6]: to drop

**Review by qdp-odoo — CHANGES_REQUESTED** [review]
forgot to check the first ones. Now it should be ok

**Comment by moerradi** [comment]
Hahah it sounded logical but i didn't do this manually i created a script for it XD

**Comment by moerradi** [comment]
done

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**